python/lsst/obs/test/dualRawImage.py

```diff
@@ -20,6 +20,8 @@
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
 #
 
+__all__ = ("RawAndFlatContainer", "RawAndFlatAssembler", "RawAndFlatDisassembler")
```

Comment:
It looks like the other packages use a list for `__all__` , so I would recommend a list here for consistency.